.github/workflows/blade-chromatic.yml

```diff
@@ -0,0 +1,35 @@
+# Workflow name
+name: Chromatic
+
+# Event for the workflow
+on: push
+
+env:
+  GITHUB_ACCESS_TOKEN: ${{ secrets.CI_BOT_TOKEN }}
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    name: Chromatic Deployment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node v14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Setup Cache & Install Dependencies
+        uses: bahmutov/npm-install@v1
+        with:
+          install-command: yarn --frozen-lockfile
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          workingDir: packages/blade
+          buildScriptName: react:storybook:build
+          exitOnceUploaded: true
```

Comment:
Wondering why this is `false` by default
https://github.com/chromaui/action/blob/main/action.yml#L84-L86

Reply:
Otherwise the deployment fails and CI check fails as well when there's a change.